modules/deployments-ab-testing-lb.adoc

```diff
@@ -205,7 +205,7 @@ Not all routers may support multiple or weighted backends.
 +
 [source,terminal]
 ----
-$ oc new-app openshift/deployment-example --name=ab-example-a
+$ oc new-app openshift/deployment-example --name=ab-example-a --as-deployment-config=true --labels=ab-example=true --env=SUBTITLE\=shardA
```

Comment:
The ` oc new-app` will create a service by default , but we never use it . To avoid confusion， we need to delete the default svc : 
`oc delete svc/ab-example-a`